### PR TITLE
new website + disable certificate check

### LINKED ISSUE #24: Update Math Genealogy Project URL
As mentioned in [this comment](https://github.com/davidalber/geneagrapher/issues/22#issuecomment-653797962) of #22, the Math Genealogy Project website moved. The URL in Geneagrapher needs to be updated and SSL verification issues overcome.

### PR DESCRIPTION
**Pull request by schoeps**
fixes #24 (but disabling secure connections)

### CODE REVIEW

**Comment by schoeps** [comment]
Strange... it stopped working. Resulting dot files are empty :(

File: geneagrapher/grabber.py [hunk 1]
```diff
@@ -24,8 +25,8 @@ def get_record(self, id):
         advisor ids, the mathematician name, the mathematician
         institution, and the year of the mathematician's degree.
         """
-        url = "http://genealogy.math.ndsu.nodak.edu/id.php?id=" + str(id)
```
Comment (davidalber) [inline geneagrapher/grabber.py:27]: Note: the genealogy.math.ndsu.nodak.edu domain works again, but it's an alias to mathgenealogy.org.

```
$ host genealogy.math.ndsu.nodak.edu
genealogy.math.ndsu.nodak.edu is an alias for mathgenealogy.org.
mathgenealogy.org has address 66.42.119.162
mathgenealogy.org mail is handled by 0 mail.mathgenealogy.org.

$ host mathgenealogy.org
mathgenealogy.org has address 66.42.119.162
mathgenealogy.org mail is handled by 0 mail.mathgenealogy.org.
```

File: geneagrapher/grabber.py [hunk 2]
```diff
@@ -24,8 +25,8 @@ def get_record(self, id):
...
-        page = urllib.request.urlopen(url)
+        url = "http://mathgenealogy.org/id.php?id=" + str(id)
+        page = urllib.request.urlopen(url, context=ssl._create_unverified_context())
```
Comment (davidalber) [inline geneagrapher/grabber.py:29]: I still don't like this, but I believe it's misconfiguration on the server end. Here's some feedback from a couple sites that verify SSL certificates:

From https://www.sslshopper.com/ssl-checker.html#hostname=mathgenealogy.org:

image

From https://www.digicert.com/help/:

image

So I think we don't have any alternative at the moment.

**Comment by davidalber** [comment]
Thank you!

I'll need to spin up a new release after some testing, updating the requirements, and so on.

**Pull request merged**